virtcontainers/hyperstart_agent.go

```diff
@@ -798,3 +798,8 @@ func (h *hyper) sendCmd(proxyCmd hyperstartProxyCmd) (interface{}, error) {
 
 	return h.client.HyperWithTokens(proxyCmd.cmd, tokens, proxyCmd.message)
 }
+
+func (h *hyper) onlineCPUMem() error {
+	// hyper agent onlines vCPUs and Memory automatically
```

Comment:
Well, that is not true. hyperstart supports `ONLINECPUMEM` cmd to online cpu and memory.